src/cli_commands.h

```diff
@@ -2,8 +2,8 @@
  * It contains alternative structs which omit the parts of the commands table
  * that are not suitable for redis-cli, e.g. the command proc. */
```

Comment:
```suggestion
 * that are not suitable for valkey-cli, e.g. the command proc. */
```

Reply:
@hwware Let's leave it for another PR and we change only one thing in each PR. It is faster and more clear.